portfolio/src/main/webapp/script.js

```diff
@@ -111,18 +143,23 @@ function setGalleryEvents(galleryPics) {
 
 
 function setUp() {
```

Comment:
nit: One code style I like and highly recommend is to change the order of functions so that you call them before you define them. In JavaScript, functions and vars are hoisted. What this means is that regardless of where you put them, the Java interpreter will always read the definitions before executing any code, so they'll be available before they are written. This allows you to write code that reads from the top-down just like a book.

Example:
```
function setUp() {
  setTabEvents();
  setGalleryEvents();
}

function setTabEvents() {
  // ...
}

function setGalleryEvents() {
  // ...
}
```

Reply:
Right! I completely forgot about that. I'll reorganize the functions once everything is merged.